src/test/java/org/folio/di/StubTest.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.di;
+
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(VertxUnitRunner.class)
+public class StubTest extends  DiAbstractRestTest {
```

Comment:
Will this class be removed later?

Reply:
It was stub test. Now it was changed to the original test.